test/image/mocks/zzz-automargin-title-paper.json

```diff
@@ -0,0 +1,29 @@
+{
+    "data": [
+        {
+            "showlegend": false,
+            "type": "scatter",
+            "x": [
+            1,
+            2,
+            3
+            ],
+            "y": [
+            4,
+            5,
+            6
+            ]
+        }],
+    "layout": {
+        "height": 300,
+        "width": 400,
+        "margin": {"t":0, "b": 0, "l": 0, "r": 0},
+        "title": {
+            "automargin": true, 
+            "text": "Basic title",
+            "font": {"size": 24},
+            "pad": {"t": 15, "b": 10},
+            "yref": "paper"
```

Comment:
Also please add a new mock exactly like this one; but using `y: 0`.
Thanks again.